configuration/wireless/access-point-bridged.md

```diff
@@ -0,0 +1,157 @@
+
+# Setting up a Raspberry Pi as a Bridged Wireless Access Point
+
+The Raspberry Pi can be used as a bridged wireless access point within an existing ethernet network. This will extend the network to wireless computers and devices.
+
+If you wish to create a wireless network that could function stand-alone, consider instead setting up a [routed access point.](./access-point-routed.md)
+
+```
+                                         +- RPi -------+
+                                     +---+ Bridge      |
+                                     |   | WiFi AP     +-)))
+                                     |   | 192.168.1.2 |         +- Laptop ----+
+                                     |   +-------------+     (((-+ WiFi STA    |
+                 +- Router ----+     |                           | 192.168.1.5 |
+                 | Firewall    |     |   +- PC#2 ------+         +-------------+
+(Internet)---WAN-+ DHCP server +-LAN-+---+ 192.168.1.3 |
+                 | 192.168.1.1 |     |   +-------------+
+                 +-------------+     |
+                                     |   +- PC#1 ------+
+                                     +---+ 192.168.1.4 |
+                                         +-------------+
+```
+
+This can be done using the inbuilt wireless features of the Raspberry Pi 4, Raspberry Pi 3 or Raspberry Pi Zero W, or by using a suitable USB wireless dongle that supports access point mode.
+It is possible that some USB dongles may need slight changes to their settings. If you are having trouble with a USB wireless dongle, please check the forums.
+
+This documentation was tested on a Raspberry Pi 3B running a factory installation of Raspbian Buster Lite (Jul. 2019). 
```

Comment:
IMHO 'Jul.' should be expanded to 'July', and perhaps "a fresh installation" makes more sense than "a factory installation" ?

Reply:
I'd rather it just said the latest Raspbian image. Otherwise it's an extra place that needs to be updated each time an image comes out.

Reply:
Probably still worth specifying "Raspbian Buster" though?

Reply:
I think so, yes. The "netfilter-persistent" thing is likely to blow in the face of people using older releases.

Reply:
> IMHO 'Jul.' should be expanded to 'July', and perhaps "a fresh installation" makes more sense than "a factory installation" ?

Yes. fresh install is better